test/AndcultureCode.CSharp.Sitefinity.Core.Tests/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,20 @@
+using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+[assembly: AssemblyTitle("UnitTestProject1")]
```

Comment:
Not sure this is a big deal (being the test project and all), but this looks like a generated/default assembly name.